app/proxy_pages.rb

```diff
@@ -0,0 +1,110 @@
+class ProxyPages
+  def self.resources
+    publishing_api_docs +
+      govuk_schema_names +
+      app_docs +
+      app_docs_json +
+      document_types +
+      supertypes
+  end
+
+  def self.publishing_api_docs
+    PublishingApiDocs.pages.map do |page|
+      {
+        url: "/apis/publishing-api/#{page.filename}.html",
```

Comment:
```suggestion
        path: "/apis/publishing-api/#{page.filename}.html",
```

I think this is better named `path` as it's not a full URL.

Reply:
Thanks - have now updated 👍